build.gradle

```diff
@@ -1,9 +1,5 @@
-task wrapper(type: Wrapper) {
```

Comment:
We don't need this task any more, plus it stuck us to an outdated Gradle version.